uploader/tagged.go

```diff
@@ -49,6 +52,61 @@ func urlParse(rawurl string) (*url.URL, error) {
 	return m, err
 }
 
+func (u *Tagged) parseName(name string, days uint16,
+	// reusable buffers
+	tag1 []string, wb *RowBinary.WriteBuffer, tagsBuf *RowBinary.WriteBuffer) error {
+
+	m, err := urlParse(name)
+	if err != nil {
+		return err
+	}
+
+	version := uint32(time.Now().Unix())
+
+	wb.Reset()
+	tagsBuf.Reset()
+	tag1 = tag1[:0]
+
+	mName := m.Path
```

Comment:
It's not crucial at all, just a side note. I don't see a reason to allocate a new local var mName, we can use m.Path

Reply:
Also fix my mistake: tagsBuf appended during tags scan, so need be checked after all steps.